testing/docker.go

```diff
@@ -64,7 +63,7 @@ type DockerContainer struct {
 	Cmd                []string
 	ContainerId        string
 	ContainerName      string
-	ContainerJSON      dockertypes.ContainerJSON
+	ContainerJSON      dockercontainer.InspectResponse
```

Comment:
Can you back out these changes? The `testing` package is deprecated in favor of `dktest`. This is only kept in case users were still using it.

Reply:
this causes linter red

Reply:
just reverted and got this:
![image](https://github.com/user-attachments/assets/c08de336-fc61-41d5-9ae5-70942e4864d7)

Reply:
You can use `//nolint:staticcheck` to ignore these false positives for now. I'm not sure what's different in CI since I'm not getting these warnings locally.

Reply:
downgrading clickhouse-go/v2 version then, because it was needed by clickhouse/v2

Reply:
not possible all broken if downgrading

Reply:
@kokizzu Use [clickhouse v2.32.2](https://github.com/ClickHouse/clickhouse-go/blob/v2.32.2/go.mod#L11) to stay on docker v27

Reply:
@dhui cannot, all related to clickhouse broken if downgrading to that version